docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -242,6 +242,30 @@ for example::
       ansible.builtin.set_fact:
         myvar: "{{ result.stdout | from_json }}"
 
+.. versionadded:: '2.12'
+
+To remove preceding and trailing unwanted lines from the string that possibly contains JSON, use the 'clean_json' filter.
```

Comment:
In the example below, the filter displays everything between the opening `{` and the closing `}` - if that is always the behavior, I would document it that way:
```suggestion
To remove preceding and trailing lines from any string that might contain JSON, returning only the data between the opening ``{`` and the closing ``}``, use the 'clean_json' filter.
```

Reply:
it also handles `[ ]` , originally i mention 'JSON object { } ' and 'JSON list [ ]', I'll put that back